source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,346 @@ Sorts Builders
+the following documents:
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+
+Sorting Direction
+-----------------
+
+The ``Sorts`` class provides two methods for specifying the sort order of data
+that does not contain any duplicate values. Data can be sorted from smallest to largest 
+with ``Sorts.ascending()``, and data can be sorted from largest to smallest with
+``Sorts.descending()``. The following two subsections provide examples
+on how to use these two methods. 
+
+Ascending
+~~~~~~~~~
+
+To specify an ascending sort, use the ``Sorts.ascending()``
+static factory method. An ascending sort returns 
+your documents from smallest to largest on the field name specified.
+
+The ``ascending`` method can be used as follows:
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+
+   collection.find().sort(ascending("<field name>"));
+
+This returns a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>`
+object containing the documents in your collection, sorted 
+from smallest to largest on the specified field name. 
+
+.. example::
+
+   In this code example, we use ``ascending`` to
+   sort the
+   :ref:`sort_example <docs_java_fundamentals_builders_sort_sort_example>`  
+   collection by ``_id``:
+
+   .. code-block:: java
+   
+      import static com.mongodb.client.model.Sorts.ascending;
+   
+      // <MongoCollection setup code here>
+      
+      Bson idSort = ascending("_id");
+      List<Document> results = new ArrayList<>();
+      collection.find().sort(idSort).into(results);
+      for (Document result : results) {
+          System.out.println(result.toJson());
+      }
+
+   The output of the above code snippet should look something like this: 
+   
+   .. code-block:: json
+   
+      {"_id": 1, "letter": "c", "food": "coffee with milk"}
+      {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+      {"_id": 3, "letter": "a", "food": "maple syrup"}
+      ...
+   
+Note that MongoDB does not guarantee sort order for documents that have
+fields with identical values. This means that if you pass "letter" to
+the ``ascending()`` method, the first document returned could be any
+of the following documents:
+
+.. code-block:: json
+
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+
+If you would like to have a guaranteed sort order 
+for documents that have fields with identical values
+you can specify additional fields to sort
+on in the event of a tie. 
```

Comment:
Let's explain in terms of "need" rather "like to have" when possible. E.g. "If you need a ..., specify additional fields..."

Reply:
Applies to all instances in the PR

Reply:
Thanks for the note. "Need" is probably a more accurate term for a customers use case, and is creates a much easier to understand sentence. Will try and remember to use this imperative language going forward. Fix:  https://github.com/mongodb/docs-java/pull/59/commits/ffd278c86dd8cd6a10320c506d2d3682a89d6810